web/ui/dashboard/package.json

```diff
@@ -12,48 +12,47 @@
 	},
 	"private": true,
 	"dependencies": {
-		"@angular/common": "^14.0.3",
-		"@angular/compiler": "^14.0.3",
-		"@angular/core": "^14.0.3",
-		"@angular/forms": "^14.0.3",
-		"@angular/platform-browser": "^14.0.3",
-		"@angular/platform-browser-dynamic": "^14.0.3",
-		"@angular/router": "^14.0.3",
-		"@auth0/angular-jwt": "^5.1.2",
-		"@markdoc/markdoc": "^0.1.12",
-		"@radix-ui/colors": "^3.0.0",
-		"@tailwindcss/container-queries": "^0.1.1",
-		"@types/markdown-it": "^12.2.3",
-		"@types/react": "^18.0.21",
-		"axios": "^1.6.7",
-		"date-fns": "^2.27.0",
-		"monaco-editor": "^0.34.1",
-		"posthog-js": "^1.36.0",
-		"prismjs": "^1.29.0",
-		"rxjs": "^6.6.0",
-		"tslib": "^2.3.1",
-		"zone.js": "^0.11.4"
+    "@angular/common": "^14.0.3",
```

Comment:
@Oluwadaminiola can you check the changes in this file?

Reply:
This is not clear

Reply:
I did update some of the dependencies version

Reply:
Yeah, I know; I'm not sure what they do, so I want her to review them.

Reply:
there's no breaking change here @jirevwe

Reply:
Thanks. LGTM @olamilekan000